tests/infra/logging_app.py

```diff
@@ -104,6 +104,7 @@ def issue(
         msg=None,
         user=None,
         url_suffix=None,
+        private_url=None,
```

Comment:
I think this should be called `url`, or in fact `uri` in this case as this doesn't include the full address?

Reply:
Just went with the existing convention in the current test i.e. `url`